ERDDAP/cache_ERDDAP.py

```diff
@@ -77,6 +92,10 @@ def create_table_from_schema(pg_engine, table_name, schema_file, pg_schema='publ
             sql = Path(schema_file).read_text()
             pg_conn.execute(text(sql))
 
+        print(f"Adding geom column")
```

Comment:
You'll need to switch the order of operations here - since the update query would be happening before the alter table query.  Verify that `geom` exists first and then perform the update.

Also, as far as I know, you should only need to call `commit` once you're done running all of your queries, that should happen at the end (or if there's been an error, `rollback` to cancel the transaction).